src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -58,16 +63,6 @@ export class BuilderCanvas extends Vue {
     console.debug('watch/activeFlow.blocks:', 'blocks inside active flow have changed, validating ...')
     this.validate_allBlocksWithinFlow()
   }, DEBOUNCE_VALIDATION_TIMER_MS)
-
-  @Watch('activeFlow.resources', {deep: true, immediate: true})
-  async onResourcesOnActiveFlowChanged(newResources: IResources, oldResources: IResources): Promise<void> {
-    console.debug('watch/activeFlow.resources:', 'resources inside active flow have changed, validating ...')
-    await this.validate_resourcesOnSupportedValues({
```

Comment:
We don't need to watch resources anymore, because it's inside the flow (since RC4), therefore I moved this `validate_resourcesOnSupportedValues` inside `debounceFlowValidation`